### ISSUE TITLE
Where is the forum component?

### ISSUE THREAD

**Issue by erondpowell** [CodyBontecou/blog#56]
Cody, 

Just read your Home page. 你中文咋样？
I also lived in Yantai, then Qingdao for 5 years.

Anyway, I wanted to ask where are you hooking in the forum component? 
I took a look through the layout and config.js and couldn't easily identify it.

**Comment by CodyBontecou** [comment]
Hey Erond!

My Chinese is... lacking 🥲 

Which component exactly are you asking about? The commenting one? That can be seen here - https://github.com/CodyBontecou/blog/blob/main/src/.vitepress/components/Comments.vue

**Comment by erondpowell** [comment]
Yes, you have a comment/forum widget. Im wondering where you hook it in in vitepress. I use vitepress for work, and want to add a fairly complicated forum into the documentation and wanna learn from a few application instances.

**Comment by erondpowell** [comment]
I couldn't find easily find the Layout (or wherever) you hooked the forum component into 🐒

**Comment by CodyBontecou** [comment]
Hey @erondpowell. This takes place in the [MyLayout](https://github.com/CodyBontecou/blog/blob/main/src/.vitepress/theme/MyLayout.vue) component.
